packages/@vivliostyle/theme-base/css/common/meta-properties.css

```
@@ -0,0 +1,56 @@
+:root {
+  /**
+   * Typography
+   */
+  --vs-font-family: sans-serif;
+  --vs-font-feature-settings: 'kern', 'liga', 'calt';
+  --vs-font-size: 16px;
+  --vs-font-size-smaller: calc(var(--vs-font-size) * 0.8);
+  --vs-font-style: normal;
+  --vs-font-weight-normal: 400;
+  --vs-font-weight-bold: 700;
+  --vs-hanging-punctuation: allow-end;
+  --vs-hyphens: auto;
+  --vs-line-break: strict;
+  --vs-line-height: 1.5;
+  --vs-letter-spacing: 0;
+  --vs-overflow-wrap: break-word;
```

`overflow-wrap: break-word` では分割できない長い単語がテーブルセル内にあった場合に役に立たないことがあります。それを解決するのに `overflow-wrap: anywhere` のほうがよいと思います。

```suggestion
  --vs-overflow-wrap: anywhere;
```

---

デフォルトのoverflow-wrapを `anywhere` に変更しました https://github.com/vivliostyle/themes/blob/c70e5fe79f972b34a17d390a22107052ae69eeda/packages/%40vivliostyle/theme-base/css/common/meta-properties.css#L23